aicsimageio/readers/reader.py

```diff
@@ -834,6 +834,212 @@ def mosaic_tile_dims(self) -> Optional[Dimensions]:
 
         return None
 
+    def generate_stack(
+        self,
+        mode: str,
+        drop_non_matching_scenes: bool = False,
+        select_scenes: Optional[
+            Union[List[Union[str, int]], Tuple[Union[str, int], ...]]
+        ] = None,
+        scene_character: str = "I",
```

Comment:
does this really need to be a user option?  Can we just say we're using "I" and call it good?